core/src/main/java/zipkin2/finagle/SpanRecorder.java

```diff
@@ -27,13 +27,14 @@
 import java.net.InetSocketAddress;
 import java.util.Iterator;
 import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.TimeUnit;
 import scala.runtime.BoxedUnit;
 import zipkin2.reporter.Reporter;
 import zipkin2.v1.V1SpanConverter;
 
 final class SpanRecorder extends AbstractClosable {
   private static final String ERROR_FORMAT = "%s: %s"; // annotation: errorMessage
-  final Duration ttl = Duration.fromSeconds(120);
+  final Duration ttl = Duration.apply(120, TimeUnit.SECONDS);
```

Comment:
there are some warnings like this I will try to make all go away, but will wait to see if my thinking about 2.11 will work before digging too much